docs/source/start.ipynb

```diff
@@ -333,6 +333,67 @@
     "df.ww"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Woodwork with a Series\n",
+    "\n",
+    "Woodwork also can be used to store typing information on a Series. There are two approaches for initializing Woodwork on a Series. If you have a Series that has a dtype that is compatible with the specified or inferred LogicalType, Woodwork can be initialized through the `ww` namespace, just as with DataFrames."
```

Comment:
I wonder if we should be more explicit about what it means for a dtype to be "compatible" with a Logical Type.

Reply:
I get your point here. Trying to think about how we can do that cleanly without launching into too much of a detailed explanation here, as that should be covered in the Understanding Types and Tags guide. Maybe it would be sufficient to refer users to that guide if they need more information?

Reply:
Yeah, maybe a quick reference to the idea that the Series' dtype has to match the LogicalType's physical type, and if they need more information reference the types and tags guide